src/commands/login-new.js

```diff
@@ -125,10 +131,17 @@ class LoginNewCommand extends ApifyCommand {
             consoleUrl.searchParams.set('localCliComputerName', encodeURIComponent(computerName()));
 
             outputs.info(`Opening Apify Console at "${consoleUrl.href}"...`);
+            outputs.info('You can also insert token manually below');
```

Comment:
Maybe this could be `You can also paste your Apify token below`?

Reply:
You are natural born copywriter 😄 Fixed